lib/errors.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+#
+module XmlJats
+  class DoctypeError < RuntimeError
```

Comment:
Hopefully for all these bespoke errors, we can set a default message (some examples found [here](http://blog.honeybadger.io/ruby-custom-exceptions/)) so that messages dont need to be defined [here](https://github.com/growkudos/xml_jats_resolver/pull/1/commits/74be2993f7e4ef9819c92304c73b533506d23308#diff-7cffadcfcaedd4f3d3d9b7ef50dd7ab0R43) and [here](https://github.com/growkudos/xml_jats_resolver/pull/1/commits/74be2993f7e4ef9819c92304c73b533506d23308#diff-7cffadcfcaedd4f3d3d9b7ef50dd7ab0R46) and [here](https://github.com/growkudos/xml_jats_resolver/pull/1/commits/74be2993f7e4ef9819c92304c73b533506d23308#diff-7cffadcfcaedd4f3d3d9b7ef50dd7ab0R53). :)